<launch>
    <!-- Parameters -->
    <!-- Conda env -->
    <param name="conda_env_name" value="happypose"/>
    <!-- Dataset (hope/tless/ycbv) -->
    <param name="dataset" value="ycbv"/>
    <!-- Camera -->
    <arg name="cam_input" default="/azure/rgb/image_raw" />
    <arg name="cam_info" default="/azure/rgb/camera_info" />
    <!-- Detectron -->
    <arg name="detectron_result" default="/detectron2_ros_specific/result" />

    <!-- Node -->
    <node name="cosypose_server" pkg="happypose_ros1" type="cosypose_server.py" output="screen">
        <param name="conda_env_name" value="$(arg conda_env_name)"/>
        <param name="dataset" value="$(arg dataset)"/>
        <param name="cam_input" value="$(arg cam_input)"/>
        <param name="cam_info" value="$(arg cam_info)"/>
        <param name="detectron_result" value="$(arg detectron_result)"/>
    </node>
</launch>
